{"identifier":{"url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/LocalizedError-Implementations"]]},"sections":[],"metadata":{"role":"symbol","extendedModule":"OpenAI","roleHeading":"Instance Property","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?"}],"modules":[{"name":"OpenAI"}],"externalID":"s:6OpenAI8APIErrorV16errorDescriptionSSSgvp","symbolKind":"property","title":"errorDescription"},"schemaVersion":{"major":0,"minor":3,"patch":0},"variants":[{"paths":["\/documentation\/openai\/apierror\/errordescription"],"traits":[{"interfaceLanguage":"swift"}]}],"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}],"kind":"declarations"}],"kind":"symbol","abstract":[{"text":"Inherited from ","type":"text"},{"code":"LocalizedError.errorDescription","type":"codeVoice"},{"text":".","type":"text"}],"references":{"doc://OpenAI/documentation/OpenAI/APIError/LocalizedError-Implementations":{"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/LocalizedError-Implementations","url":"\/documentation\/openai\/apierror\/localizederror-implementations","abstract":[],"type":"topic","kind":"article","title":"LocalizedError Implementations","role":"collectionGroup"},"doc://OpenAI/documentation/OpenAI":{"type":"topic","kind":"symbol","url":"\/documentation\/openai","role":"collection","title":"OpenAI","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","abstract":[]},"doc://OpenAI/documentation/OpenAI/APIError":{"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"APIError"}],"title":"APIError","navigatorTitle":[{"text":"APIError","kind":"identifier"}],"type":"topic","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError","abstract":[],"kind":"symbol","role":"symbol","url":"\/documentation\/openai\/apierror"},"doc://OpenAI/documentation/OpenAI/APIError/errorDescription":{"url":"\/documentation\/openai\/apierror\/errordescription","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"role":"symbol","abstract":[],"kind":"symbol","title":"errorDescription","type":"topic","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIError\/errorDescription"}}}